spec/features/user_show_basic_elements_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly make sure that all tests for your views are passed
![image](https://github.com/maximoortelli/Blog_App/assets/24703171/9c5a0bec-cec9-4fe0-a553-344c42a88622)
Currently, there are 30 failed tests.